### PR TITLE
Wick

### PR DESCRIPTION
**Pull request by BeeXD**

### CODE REVIEW

File: websites/W/Wick/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      "name": "BeeXD",
+      "id": "745199026021859338"
+    },
+    "service": "Wick Dashboard",
+    "description": {
+      "en": "Wick Bot Dashboard"
+    },
+    "url": "wickbot.com",
+    "version": "1.0.0",
+    "logo": "https://wickbot.com/wick.png",
```
Comment (EncryptedDev) [inline websites/W/Wick/dist/metadata.json:13]: Imgur only please

File: websites/W/Wick/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "author": {
+      "name": "BeeXD",
+      "id": "745199026021859338"
+    },
+    "service": "Wick Dashboard",
+    "description": {
+      "en": "Wick Bot Dashboard"
```
Comment (EncryptedDev) [inline websites/W/Wick/dist/metadata.json:9]: I doubt that is the proper description

File: websites/W/Wick/presence.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
+const presence = new Presence({
+  clientId: "890182436695187528"
+}),
+
```
Comment (EncryptedDev) [inline websites/W/Wick/presence.ts:4]: Remove line

File: websites/W/Wick/presence.ts [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+ browsingStamp = Math.floor(Date.now() / 1000);
+
+let title: HTMLElement;
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "wick"
+  };
+
+  if (document.location.hostname === "wickbot.com") {
```
Comment (EncryptedDev) [inline websites/W/Wick/presence.ts:14]: Not needed

File: websites/W/Wick/presence.ts [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+      presenceData.details = "Reading the TOS";
+      presenceData.smallImageKey = "reading";
+    } else if (document.location.pathname.includes("/status")) 
+      presenceData.details = "Viewing the status";
+     else if (document.location.pathname.includes("/premium")) 
+      presenceData.details = "Viewing Wick Premium Plans";
+    
+  }
+
+  if (presenceData.details === null) {
```
Comment (EncryptedDev) [inline websites/W/Wick/presence.ts:41]:
```suggestion
  if (!presenceData.details) {
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
Not much to the presence, please support more pages (if applicable) and use more DOM selectors

**Comment by BeeXD** [comment]
The wick bot dash board doesnt have much pages .

**Comment by EncryptedDev** [comment]
> The wick bot dash board doesnt have much pages .
Unfortunately that isn't good enough, the presence is low quality and cannot be merged.

**Pull request closed without merging**